staging/src/k8s.io/apiserver/pkg/server/options/authorizationconfig/metrics/metrics.go

```diff
@@ -38,7 +38,7 @@ var (
 			Subsystem:      subsystem,
 			Name:           "automatic_reloads_total",
 			Help:           "Total number of automatic reloads of authorization configuration split by status and apiserver identity.",
-			StabilityLevel: metrics.ALPHA,
```

Comment:
While this FG is GA now, metrics should touch all stages when graduating, as such this should soak in BETA for atleast one release.